src/pages/recipes/{ContentfulRecipe.title}.tsx

```diff
@@ -1,74 +1,206 @@
 import React from 'react';
 
-import { graphql, PageProps } from 'gatsby';
-import { Helmet } from 'react-helmet';
+import { ChevronLeftIcon } from '@chakra-ui/icons';
+import { Flex, Heading, IconButton, Text, Image, HStack, VStack, Box } from '@chakra-ui/react';
+import styled from '@emotion/styled';
+import { graphql, Link, PageProps } from 'gatsby';
+import TitledList from 'src/components/TitledList';
 
 import Layout from '../../components/Layout';
 
 interface Props extends PageProps {
   data: GatsbyTypes.RecipeByTitleQuery;
 }
 
+const CenterImage = styled(Image)`
+  width: 100%;
+`;
+const Container = styled(Box)`
+  padding: 0px 16px 20px 16px;
+`;
+const Ingred = styled(Text)`
+  padding: 30px 0px 10px 0px;
+  font-size: 18px;
+`;
+const Title = styled(Heading)`
+  font-size: 18px;
+`;
+const Preparation = styled(Text)`
+  padding: 15px 0px 0px 0px;
+`;
+
+const ListOfIngreds = styled(Text)`
+  font-size: 16px;
+`;
+
 function RecipeTemplate(props: Props): JSX.Element {
   const recipe = props.data.contentfulRecipe;
-  const siteTitle = props.data.site?.siteMetadata?.title;
+
+  const totalTime =
+    recipe?.prepTime && recipe?.totalTime ? recipe?.prepTime + recipe?.totalTime : '';
+
+  const prepDirections = recipe?.prepDirections?.prepDirections
+    ?.split('\n')
+    .map(prepDir => prepDir.slice(2));
+
+  const instructions = recipe?.directions?.directions
+    ?.split('\n')
+    .map(direction => direction.slice(2));
+
+  const notes = recipe?.notes?.notes?.split('\n').map(note => note.slice(2));
+
+  const ingredients = recipe?.ingredients?.ingredients?.split('\n');
+  const ingredientGroups = [];
+
+  if (ingredients) {
+    let currentIndex = 0;
+
+    while (currentIndex < ingredients.length) {
+      const currentGroup = [];
+      const amountIndex = currentIndex + 1;
+      const notesIndex = currentIndex + 2;
+      currentGroup.push(ingredients[currentIndex].slice(2));
+      currentGroup.push(ingredients[amountIndex].slice(4));
+
+      const hasNotes = notesIndex < ingredients.length && ingredients[notesIndex][0] == ' ';
+
+      if (hasNotes) {
+        currentGroup.push(ingredients[notesIndex].slice(4));
+        currentIndex++;
+      }
+
+      ingredientGroups.push(currentGroup);
+      currentIndex += 2;
+    }
+  }
+
+  const isMobile = false;
```

Comment:
Rather than using some state check, we should try to leverage responsivity when we can: https://chakra-ui.com/docs/features/responsive-styles